webapp/src/components/PageHeader.tsx

```diff
@@ -186,10 +193,18 @@ const PageHeader = () => {
                 },
               }}
             >
-              <Settings />
+              <SettingsIcon />
             </IconButton>
             <HelpMenu />
           </Box>
+          <Dialog
+            aria-labelledby="config-dialog-title"
+            maxWidth="md"
+            scroll="paper"
+            open={openConfigModal}
+          >
+            <Settings onClose={setOpenConfigModal} />
```

Comment:
Don't you have an error, here?

Reply:
Yes, I do, as we pass the boolean value to it. I am working on it to fix that.

Reply:
You just added back the unnecessary boolean argument that I asked you to remove [here](https://github.com/ServiceNow/azimuth/pull/437#discussion_r1131849905)? The right solution is rather as follows, just like you did for the `IconButton`'s `onClick`:
```suggestion
            <Settings onClose={() => setOpenConfigModal(false)} />
```

Reply:
Yes, that error message due to that change was blocking me to apply other changes while I figure out the solution. So, I reverted back and working on it now.

Reply:
Applied the changes to remove the boolean property in my recent commit. Thank you for the suggestion.